backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/repository/GitMacheteRepository.java

```diff
@@ -0,0 +1,70 @@
+package com.virtuslab.gitmachete.backend.impl.repository;
+
+import com.jcabi.aspects.Loggable;
+import io.vavr.collection.Set;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.branchlayout.api.BranchLayout;
+import com.virtuslab.gitcore.api.GitCoreException;
+import com.virtuslab.gitcore.api.IGitCoreRepository;
+import com.virtuslab.gitmachete.backend.api.GitMacheteException;
+import com.virtuslab.gitmachete.backend.api.IGitMacheteRepository;
+import com.virtuslab.gitmachete.backend.api.IGitMacheteRepositorySnapshot;
+import com.virtuslab.gitmachete.backend.api.ILocalBranchReference;
+import com.virtuslab.gitmachete.backend.impl.aux.Aux;
+import com.virtuslab.gitmachete.backend.impl.aux.CreateGitMacheteRepositoryAux;
+import com.virtuslab.gitmachete.backend.impl.aux.DiscoverGitMacheteRepositoryAux;
+import com.virtuslab.gitmachete.backend.impl.hooks.PreRebaseHookExecutor;
+import com.virtuslab.gitmachete.backend.impl.hooks.StatusBranchHookExecutor;
+import com.virtuslab.qual.guieffect.UIThreadUnsafe;
+
+@RequiredArgsConstructor
+public class GitMacheteRepository implements IGitMacheteRepository {
```

Comment:
I dropped `@CustomLog` here. 
however there are some `@Loggable` below in this file.
`LOG` calls were from Aux methods only.

is it ok?

Reply:
Apparently [`@Loggable`](https://aspects.jcabi.com/apidocs-0.22.6/com/jcabi/aspects/Loggable.html) takes care of providing a correct SLF4J logger... so it seems it's okay